tests/PluginTest.php

```diff
@@ -160,6 +163,31 @@ public function testGetSubscribedEvents()
         $this->assertInternalType('array', $plugin->getSubscribedEvents());
     }
 
+    public function testGetAlphabetizedList()
+    {
+        // We have to set the private method accessible
+        $method = new \ReflectionMethod(
+            '\Phergie\Irc\Plugin\React\CommandHelp\Plugin', 'alphabetize'
+        );
+        $method->setAccessible(TRUE);
+
+        //Test our method with a few plugins
+        $this->assertEquals(
+            'bofh tableflip',
```

Comment:
Still working on that bit.  I was just able to run tests for the package and, I believe mock a plugin. I'm slowly getting there and will remove the bits I needed to test manually.  Not being able to even run the test suite was an inconvenience.